counsel.el

```diff
@@ -4090,15 +4100,21 @@ candidate."
   (org-map-entries
    (lambda ()
      (let* ((components (org-heading-components))
-            (level (make-string
-                    (if org-odd-levels-only
-                        (nth 1 components)
-                      (nth 0 components))
-                    ?*))
-            (todo (nth 2 components))
-            (priority (nth 3 components))
+            (level (when (eq counsel-org-goto-display-style 'headline)
+                     (make-string
+                      (if org-odd-levels-only
+                          (nth 1 components)
+                        (nth 0 components))
```

Comment:
I'd write this as `(nth (if org-odd-levels-only 1 0) components)`, but maybe that's just me.

Reply:
These lines were just reindented. I prefer to hold back on refactoring for a separate commit.